process/interceptors/factory/metaInterceptedDataFactory_test.go

```diff
@@ -126,6 +126,23 @@ func TestMetaInterceptedDataFactory_CreateInterceptedShardHdrShouldWork(t *testi
 	assert.True(t, ok)
 }
 
+func TestMetaInterceptedDataFactory_CreateInterceptedMetaHdrShouldWork(t *testing.T) {
+	t.Parallel()
+
+	marshalizer := &mock.MarshalizerMock{}
+	emptyHdr := &block.Header{}
+	emptyHdrBuff, _ := marshalizer.Marshal(emptyHdr)
+
+	midf, _ := factory.NewMetaInterceptedDataFactory(createMockMetaArgument(), factory.InterceptedMetaHeader)
+
```

Comment:
remove empty line.

Reply:
Removed line 135 as to respect the 3 main parts of a test func: arrange, act and assert